koku/api/report/test/ocp/test_views.py

```diff
@@ -1245,6 +1245,49 @@ def test_execute_query_with_group_by_order_by_and_limit(self):
                     self.assertTrue(current_value <= previous_value)
                     previous_value = current_value
 
+    def test_execute_query_with_group_by_project_order_by_distributed_cost(self):
+        """Test that data is grouped by and limited on order by."""
+
+        def get_distributed_cost(dikt):
+            return dikt.get("values")[0].get("cost", {}).get("distributed", {}).get("value")
+
+        for order_option in ["asc", "desc"]:
```

Comment:
Unittest makes it a bit cumbersome to test different scenarios like this. While it would duplicate the setup code, I think it would be cleaner to have distinct tests for ascending and descending cases. This makes a failure of either scenario much easier to identify and troubleshoot.

Reply:
Good point, let me throw the order option in a subTest so that its treated as two different tests.